src/engine/read_json.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include "json.hpp"
+using json = nlohmann::json;
+
+/* Example json format
+{
```

Comment:
Surround stuff like this will the markdown ` ` ` things, will render nicely if we ever generate html